### ISSUE TITLE
[WIP] Test Cases

### ISSUE THREAD

**Issue by sasicodes** [lenstube-xyz/lenstube#579]
- [ ] Auth
- [ ] Channel Settings
- [ ] Explore
- [ ] Bytes
- [ ] Subscriptions
- [ ] Video Upload - IPFS, Arweave
- [ ] Image Upload
- [ ] Metadata Upload

**Comment by zt4ff** [comment]
Here @sasicodes. It's been a while. I am back to contributing to pages.
Is this ticket open?

**Comment by sasicodes** [comment]
> Here @sasicodes. It's been a while. I am back to contributing to pages. Is this ticket open?

yes, we will restart the test cases after the v2 migration. because current test cases will break.

**Comment by zt4ff** [comment]
Hi hi @sasicodes 
what's the update on this place?

**Comment by sasicodes** [comment]
yes. need to start writing testcases.
